authors/apps/authentication/tests/test_base.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework.test import APIClient, APITestCase
+
+from ..models import User
+from ..serializers import RegistrationSerializer
+
+
+class BaseViewTest(APITestCase):
+    
+    client = APIClient()
```

Comment:
Please remove this irrelevant line as there is a client attribute on the `APITestCase` class available already.